{{cookiecutter.project_slug}}/Makefile

```diff
@@ -122,6 +126,38 @@ clean-all: clean-code clean-data clean-notebooks  ### Delete code, data, and not
 
 ##### JOBS #####
 
+.PHONY: develop
+develop: upload-code upload-notebooks  ### Run a development job
```

Comment:
Have we made any decision about hidden code uploading before a job starting?

Reply:
cc @mariyadavydova @alekseysh 
We need to make a decision!

I propose: now merge with dependencies, and then, if we decide to solve https://github.com/neuromation/cookiecutter-neuro-project/issues/118,  we can get rid of all of them at once.

Reply:
ATM I agree with @ayushkovskiy. There are too many open conflicting PRs in this project now, let's resolve them all and get back to the dependencies story.